scripts/cdn_publish.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+version="$(git describe --abbrev=0 --tags)"
+mkdir -p cdn-dist/gravity/${version}
+
+cp dist/ui-lib/*.css cdn-dist/gravity/${version}
+cp dist/ui-lib/*.js cdn-dist/gravity/${version}
+
+versionList=""
+for version in $(git tag -l --sort=-committerdate); do
+  versionList+="<li>${version}</li>"
+done
+
+file_contents=$(<./cdn-index.template)
+echo "${file_contents//\{versionlist\}/$versionList}" > cdn-dist/gravity/index.html
+
+~/.local/bin/aws s3 sync cdn-dist s3://${CDN_BUCKET} --delete --region=${PROD_BUCKET_REGION} --exclude "*" --include "*.html" --content-type "text/html; charset=utf-8"
+~/.local/bin/aws s3 sync cdn-dist s3://${CDN_BUCKET} --delete --region=${PROD_BUCKET_REGION} --exclude "*" --include "*.css" --content-type "text/css; charset=utf-8"
+~/.local/bin/aws s3 sync cdn-dist s3://${CDN_BUCKET} --delete --region=${PROD_BUCKET_REGION} --exclude "*" --include "*.js" --content-type "application/javascript; charset=utf-8"
```

Comment:
Perhaps we can tweak the `cache-control` parameters for each Gravity release's files (i.e. everything under `/gravity/${version}/`)?

Based on this article: https://csswizardry.com/2019/03/cache-control-for-civilians/ , I wonder if `immutable` could be appropriate here (signalling to browsers & proxies that they never need to revalidate those resources).

If we get it right, it could provide a small performance boost to any projects referencing these CDN files.